docs/configuration.md

```diff
@@ -194,19 +196,20 @@ spec:
 | `clusterName`                                                                                                | Set a unique cluster name to allow scoping hosts and Cluster Checks Runner easily                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| `credentials.apiKey`                                                                                         | APIKey Set this to your Datadog API key before the Agent runs. ref: https://app.datadoghq.com/account/settings#agent/kubernetes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
 | `credentials.apiKeyExistingSecret`                                                                           | APIKeyExistingSecret is DEPRECATED. In order to pass the API key through an existing secret, please consider "apiSecret" instead. If set, this parameter takes precedence over "apiKey".                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| `credentials.apiKey`                                                                                         | APIKey Set this to your Datadog API key before the Agent runs. ref: https://app.datadoghq.com/account/settings#agent/kubernetes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
For some reason the comment keeps cutting off my suggestion! ✂️ 

I'd either remove `APIKey`, and leave just, `Set this to your Datadog API key before the Agent runs. ref: https://app.datadoghq.com/account/settings#agent/kubernetes` or find a way to complete the `APIKey` portion. Something along the lines of `Your APIKey. Set this to your...`